src/components/media/controls/mediaIconButton.tsx

```diff
@@ -53,7 +55,7 @@ export function MediaIconButton(props: MediaIconButtonProps) {
     >
       <IconButton
         onClick={props.onClick}
-        className={props.className}
+        className={`${props.className ? props.className + ' ' : ''}rustic-media-icon-button`}
```

Comment:
I think the class should be added to the Icon component in line 62 instead of here